pure_pursuit_node:
  ros__parameters:
    speed: 0.4
    K_P: 0.8
    lookahead_distance: 0.3
    file_name: 'spielberg.csv'